README.md

```diff
@@ -257,7 +257,7 @@ const config = await loadConfig(configFile, cwd);
 
 ## Donators
 
-| [<img src="https://sheetjs.com/sketch128.png" width="80">](https://sheetjs.com/) | [<img src="https://raw.githubusercontent.com/fontello/fontello/main/fontello-image.svg" width="80">](https://fontello.com/) |
+| [<img src="https://sheetjs.com/sketch128.png" width="80">](https://sheetjs.com/) | [<img src="https://github.com/fontello/fontello/blob/8.0.0/fontello-image.svg" width="80">](https://fontello.com/) |
```

Comment:
Let's use https://raw.githubusercontent.com/fontello/fontello/8.0.0/fontello-image.svg

Reply:
Yes, that's better, indeed. Changed as requested.